### PR TITLE
Add Hybrid Cloud results to CPT Dashboard

### PR DESCRIPTION
**Pull request by shubham-html-css-js**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description

This PR adds Hybrid Cloud CPT data to the CPT Dashboard Home Page

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ x ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: backend/app/api/v1/endpoints/cpt/cptJobs.py [hunk 1]
```diff
@@ -37,6 +39,7 @@ async def jobs(start_date: date = Query(None, description="Start date for search
         return Response(content=json.dumps({'error': "invalid date format, start_date must be less than end_date"}), status_code=422)
 
     results = pd.DataFrame()
+    print(products.values())
```
[the remark this answers is not in the document]
Comment (shubham-html-css-js) [reply backend/app/api/v1/endpoints/cpt/cptJobs.py]: @athiruma removed

**Comment by chentex** [comment]
I have reviewed and tested the PR and it looks good and works as expected, the only thing I found out is that if for some reason the configuration or the mapper of any team fails, no data will be displayed, I am working on a solution for that, so we can include it.

**Comment by vishnuchalla** [comment]
> I have reviewed and tested the PR and it looks good and works as expected, the only thing I found out is that if for some reason the configuration or the mapper of any team fails, no data will be displayed, I am working on a solution for that, so we can include it.

Yes. We need to put [this block](https://github.com/cloud-bulldozer/cpt-dashboard/blob/main/backend/app/api/v1/endpoints/cpt/cptJobs.py#L40-L42) of code in `try...catch` so that failure in any product mapper just skips those results and moves on with renaming ones.

**Comment by athiruma** [comment]
Great Work @shubham-html-css-js  👏  .
@chentex we need to merge the patch before this PR.

**Pull request merged**